Cross.toml

```diff
@@ -7,8 +7,8 @@ passthrough = [
 [target.x86_64-unknown-linux-gnu]
 image = "ghcr.io/deislabs/akri/rust-crossbuild:x86_64-unknown-linux-gnu-0.1.16-0.0.7"
 
-[target.armv7-unknown-linux-gnueabihf]
-image = "ghcr.io/deislabs/akri/rust-crossbuild:armv7-unknown-linux-gnueabihf-0.1.16-0.0.7"
+[target.arm-unknown-linux-gnueabihf]
```

Comment:
use `armv7` here

Reply:
Fixed: https://github.com/deislabs/akri/pull/135/commits/e008c4ac1edf1acffdfebd86f084848babe741c0

Reply:
it looks like this was changed to `arm7` ... it should be `armv7` (note the `v`) ... so all instances of `arm-unknown-linux-gnueabihf` should become `armv7-unknown-linux-gnueabihf`

Reply:
Apologies, fixed: https://github.com/deislabs/akri/pull/135/commits/da8ecabe2191b29da5f18ea9724dffe7c9c4f7f7